### DISCUSSION TITLE
Automatically fill the foreign key / relation id when using the "Create" Link on the HasMany fieldtype. Just like on the BelongsTo fieldtype.

### DISCUSSION THREAD

**Discussion by andjsch** [statamic-rad-pack/runway#272] [Feature Requests]
Today, I discovered that the relation ID is only filled when using the BelongsTo fieldtype. 
See #124 for reference.

**Comment by duncanmcclean** [comment]
Hmm, this already seems to work for me: https://www.loom.com/share/c225accedea64568b7df59ef3bbd050e

Unless I'm misunderstanding you?
